docs/devs/versions.md

```diff
@@ -23,6 +23,7 @@ sMRIPrep requires niworkflows and generally must depend on one minor series of n
 
 | *sMRIPrep* series | *NiWorkflows* series | *TemplateFlow* series |
 |----|----|----|
+| 0.8.x | `~= 1.4.0` | `>= 0.7.1` |
```

Comment:
@oesteban why the strict settings for templateflow?

Reply:
This comes from the time where the version of the client would really tell you which templates were accessible.

We should probably work towards a release 1.0 of the client and pin something like `~=1.0`. WDYT?